README.md

```diff
@@ -52,3 +52,6 @@ npm install --global --production windows-build-tools
 ```
 
 [Full details here](https://www.npmjs.com/package/node-gyp 'NPM node-gyp page')
+
+## Purgecss
```

Comment:
Purge is a pretty low level utility, no need to explain in the readme.

Reply:
@erquhart I have a strongly differing view. It is something that behaves differently in the production build vs the development build, and cannot be a 100% perfect optimisation. Under some scenarios CSS will be deleted that should not be, and in that case, it's extremely hard as the developer to track down the issue unless you realise PurgeCSS is the source. From my perspective, one of the greatest things about React is that it makes applications easier to reason about. PurgeCSS goes in the opposite direction, it makes the app harder to reason about. That's why I'd strongly favour keeping the notice.

Reply:
I think @chmac has a point in a sense that gatsby plugins do a lot of heavy lifting in the background, but `gatsby-config.js` has all these dependencies listed. To me this is not an issue, because i know where to look, if a new problem occurs, but there could be people new to gatsby not knowing where to look. So there's a number of options we could do. We could create a Troubleshoot.md file, we could also have a generic description about the files and folder structure and what they are responsible for in the `Readme.md` and in the same time we could add comments to the `gatsby-node.js`, explaining what each plugin does for instance, or providing a link to the github repo. Personally i would prefer having a Troubleshoot.md, we already have a Troubleshoot section for the Forms, which is in HTML, and maybe that's not the right place to have these kind of information. @erquhart What would you prefer?

Reply:
Thanks guys, great explanation - keeping the notice works for me 👍